### PR TITLE
Main 202212161220

### PR DESCRIPTION
**Pull request by fowles**

### CODE REVIEW

File: php/ext/google/protobuf/protobuf.h [hunk 1]
```diff
@@ -127,7 +127,7 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_setter, 0, 0, 1)
   ZEND_ARG_INFO(0, value)
 ZEND_END_ARG_INFO()
 
-#define PHP_PROTOBUF_VERSION "3.21.10"
```
Comment (mkruskal-google) [inline php/ext/google/protobuf/protobuf.h:130]: I wonder why this wasn't bumped to 11...

**Review by ericsalo — APPROVED** [review]

File: kokoro/macos/prepare_build_macos_rc [hunk 2]
```diff
@@ -13,7 +13,6 @@ export DEVELOPER_DIR=/Applications/Xcode_14.app/Contents/Developer
 sudo xcode-select -s "${DEVELOPER_DIR}"
 
 ##
-# Use Python 2 by default (for googletest)
```
Comment (mkruskal-google) [inline kokoro/macos/prepare_build_macos_rc:16]: nit: undo this :)

File: php/ext/google/protobuf/protobuf.h (hunk 1 above)
Comment (fowles) [reply php/ext/google/protobuf/protobuf.h:130]: I was releasing 3.21.12

Comment (fowles) [reply php/ext/google/protobuf/protobuf.h:130]: likely a merge error on 3.21.11

Comment (mkruskal-google) [reply php/ext/google/protobuf/protobuf.h:130]: Yea, but it should have been 11 in main, not 10.  It looks like we didn't merge back to main for 21.11?

Comment (fowles) [reply php/ext/google/protobuf/protobuf.h:130]: https://github.com/protocolbuffers/protobuf/pull/11210 happened

Comment (fowles) [reply php/ext/google/protobuf/protobuf.h:130]: https://github.com/protocolbuffers/protobuf/commit/032c9c8ada0587ffc41d5e635d9bb3ce2be5c16f rolled it back

**Review by mkruskal-google — APPROVED** [review]

**Review by mkruskal-google — APPROVED** [review]

**Pull request merged**